scipy/stats/tests/test_stats.py

```diff
@@ -4450,15 +4450,15 @@ def test_nist(self):
             assert_allclose(res[0], f, rtol=rtol,
                             err_msg='Failing testcase: %s' % test_case)
 
-    def test_constant_input(self):
+    @pytest.mark.parametrize("const_input, expected",[
+        (np.array([42, 42, 42]), (np.inf, 0)),
+        (np.array([3.1415, 3.1415, 3.1415]), (np.nan, np.nan))
+        ])
+    def test_constant_input(self, const_input, expected):
```

Comment:
thanks for putting into the parametrized input. It looks like you changed the test inputs though?

Can you make these `const_input` into 2 examples like before? (one with three `42`s and three `7`s, and a second one with two sets of three of `42`) so first instance is something like: 

```
@pytest.mark.parametrize("a, b, expected",[
(np.array([42, 42, 42]), np.array([7, 7, 7]), (np.inf, 0)),
```
and similar for the second test problem instance.